cmd/juju/application/showunit.go

```diff
@@ -0,0 +1,283 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application
+
+import (
+	"strings"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"github.com/juju/names/v4"
+	"github.com/juju/naturalsort"
+
+	"github.com/juju/juju/api/application"
+	"github.com/juju/juju/apiserver/params"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+const showUnitDoc = `
+The command takes deployed unit names as an argument.
+
+Optionally, relation data for only a specified endpoint
+or related unit may be shown, or just the application data. 
+
+Examples:
+    $ juju show-unit mysql/0
```

Comment:
Is this how we've done other examples? Not sure we use the $ prefix.

Reply:
In 99% of cases, we omit the $.
The one place I chose to copy from to get started since the logic is similar was show-application and that uses $. I'll fix both.